ext/drreg/drreg.c

```diff
@@ -395,7 +395,9 @@ drreg_event_bb_insert_late(void *drcontext, void *tag, instrlist_t *bb, instr_t
     instr_t *next = instr_get_next(inst);
     bool restored_for_read[DR_NUM_GPR_REGS];
     drreg_status_t res;
+    dr_pred_type_t pred = instrlist_get_auto_predicate(bb);
 
+    instrlist_set_auto_predicate(bb, DR_PRED_NONE);
```

Comment:
So all of the client instrumentation except drreg spills and restores are predicated?  This doesn't seem ideal so it should have a XXX comment and an issue number.  Ditto below.